### PR TITLE
Run Error Prone without forking compiler

### PR DESCRIPTION
**Pull request by electrum**
This allows it to benefit from Maven Daemon.

### CODE REVIEW

**Review by wendigo — APPROVED** [review]

File: .mvn/jvm.config [hunk 1]
```diff
@@ -1 +1,11 @@
--Xmx8192m 
+-Xmx8192m
+--add-exports=jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED
```
Comment (findepi) [inline .mvn/jvm.config:2]: Document these flags, as maven itself doesn't need them, nor does maven default usage.

Comment (electrum) [reply .mvn/jvm.config:2]: They are documented at https://errorprone.info/docs/installation#maven

This file does not support comments: https://github.com/apache/maven-mvnd/commit/c85f2ea28522460cd79d1918621072058d9b8996

Comment (findepi) [reply .mvn/jvm.config:2]: > They are documented at https://errorprone.info/docs/installation#maven

Yes, but looking at a maven-specific file like `.mvn/jvm.config`, it's not obvious you need to check errorprone's docs

> This file does not support comments: [apache/maven-mvnd@c85f2ea](https://github.com/apache/maven-mvnd/commit/c85f2ea28522460cd79d1918621072058d9b8996)

sad

it doesn't for mvnd, or it doesn't at all?

Comment (findepi) [reply .mvn/jvm.config:2]: > it doesn't for mvnd, or it doesn't at all?

seems neither here

https://github.com/apache/maven/blob/8852b87412c0af70efcee76f1bb12bc44635aab1/apache-maven/src/assembly/shared/init#L86

(not sure how this file is used, but at least not mvnd-specific)

Comment (electrum) [reply .mvn/jvm.config:2]: I added a comment referencing this file in the POM, explaining why we don't need to fork the compiler.

**Review by dain — APPROVED** [review]

**Pull request merged**